packages/jspsych/src/modules/plugin-api/KeyboardListenerAPI.ts

```diff
@@ -1,176 +1,149 @@
-export class KeyboardListenerAPI {
-  constructor(private areResponsesCaseSensitive: boolean, private minimumValidRt = 0) {}
+export type KeyboardListener = {
+  type: "keydown";
+  fn: (e: KeyboardEvent) => void;
+};
```

Comment:
The only type we ever use is "keydown". We could also just use the callback function directly, but it would be a breaking change (although I don't know any reason for current user code to destructure the object).

Reply:
Yeah, I can't recall why I had a type in the return value. I think we can remove it.